chcemvediet/tests/__init__.py

```diff
@@ -31,6 +32,11 @@ class CustomTestRunner(DiscoverRunner):
     def setup_test_environment(self, **kwargs):
         super(CustomTestRunner, self).setup_test_environment(**kwargs)
         settings.LANGUAGE_CODE = u'en'
+        os.environ[u'RECAPTCHA_TESTING'] = u'True'
+
+    def teardown_test_environment(self, **kwargs):
+        super(CustomTestRunner, self).teardown_test_environment(**kwargs)
+        del os.environ[u'RECAPTCHA_TESTING']
```

Comment:
@viliambalaz Tear down (a analogicky vsetky podobne upratujuce funkcie) by mali veci menit v opacnom poradi oproti setup funkciam. Predstav si to ako zatvorky. Pises `{()}` nie `{(})`. Takze kedze `RECAPTCHA_TESTING` nastavujes az po super `setup_test_environment`. Tak zmenit ho musis naspat pred super `teardown_test_environment`.

Reply:
:ok: